Objects/Objects/Other/MaterialQuantity.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+using Speckle.Core.Kits;
+using Speckle.Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Objects.Other
+{
+    public class MaterialQuantity : Base
+    {
+        public Objects.Other.Material material { get; set; }
+        public Objects.BuiltElements.Revit.Parameter volume { get; set; }
+
+        public Objects.BuiltElements.Revit.Parameter area { get; set; }
```

Comment:
Thanks for the review @teocomi 

I chose `Objects.BuiltElements.Revit.Parameter` because (1) area and volume have different units (2) there is no generic `Parameter` (3) I do not fully understand the concept of `Core.Core.Kits.Units`.

`Core.Core.Kits.Units` only covers 1d units. Or does - for instance - *feet* also imply that the units are *squarefeet* and *cubicfeet*